src/csync/csync_reconcile.cpp

```diff
@@ -47,7 +49,7 @@ static csync_file_stat_t *_csync_check_ignored(csync_s::FileMap *tree, const QBy
     if (parentlen <= 0) {
         return nullptr;
     }
-    QByteArray parentPath = path.left(parentlen);
+    auto parentPath = path.left(parentlen);
```

Comment:
I think this would be worth typing explicitly, the API is too similar and people should be able to be aware of the type for things like the no null-termination above.